packages/csshnpd/Makefile

```diff
@@ -0,0 +1,43 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=csshnpd
+PKG_VERSION:=0.1.6
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-c$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/atsign-foundation/noports/releases/download/c$(PKG_VERSION)
+PKG_HASH:=40965081365a432538fac894fe8d142dcf91de528a8ec1fb261ebfebf493488c
+
+PKG_LICENSE:=BSD-3-Clause
+PKG_LICENSE_FILES:=LICENCE
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-c$(PKG_VERSION)
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+CMAKE_OPTIONS += \
+		-S . \
+		-DBUILD_SHARED_LIBS=off \
+		-DCMAKE_C_COMPILER=gcc \
+		-DCMAKE_C_FLAGS="-Wno-calloc-transposed-args"
+
+define Package/csshnpd
+	SECTION:=net
+	CATEGORY:=Network
+	TITLE:=NoPorts Daemon
+	SELECTS:=openssh-keygen
```

Comment:
C sshnpd doesn't need ssh-keygen, because we removed the double ssh tunnel, so it doesn't generate ephemeral ssh keys anymore.

Reply:
I've removed that in https://github.com/atsign-foundation/Atsign_OpenWRT_packages/pull/13/commits/0495ab6790eed990f9540c4d164626729d437f54